addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -37,6 +38,8 @@ export const WORKER_STATE = Object.freeze({
     CONNECTING: "CONNECTING",
 });
 const MAXIMUM_RECONNECT_DELAY = 60000;
+const UUID = Date.now().toString(36) + Math.random().toString(36).substring(2);
```

Comment:
Could we use the web UUID function? 
https://github.com/odoo/odoo/blob/master/addons/web/static/src/views/utils.js#L293

Reply:
No, because it's not available in the worker bundle. Adding that file would require including many dependencies. Also, even though it's technically a pseudo-UUID (not fully reliable), it's more than sufficient for its purpose: making it easy to distinguish workers in the logs.